src/Content/Monaco.Template.Common.Application/Monaco.Template.Common.Application.csproj

```diff
@@ -23,6 +23,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\Monaco.Template.Common.Domain\Monaco.Template.Common.Domain.csproj" />
```

Comment:
You don't need this reference because it already exists by transitivity because of `Monaco.Template.Common.Infrastructure`.

```suggestion
```